lib/network_utils.pm

```diff
@@ -30,9 +30,8 @@ sub setup_static_network {
     # Set default values
     $args{ip} ||= '10.0.2.15';
     $args{gw} ||= testapi::host_ip();
-    my $dns_ip = get_required_var('STATIC_DNS_SERVER');
+    configure_static_dns(get_host_resolv_conf());
```

Comment:
https://openqa.opensuse.org/tests/962412#step/before_test/44 -> "undefined subroutine". You better move it or import.

Reply:
done